dev/unix/volta-install.sh

```diff
@@ -134,7 +134,11 @@ parse_os_info() {
         return "$exit_code"
       fi
 
-      echo "linux-openssl-$parsed_version"
+      if [ "$(uname -m)" == "arm64" ]; then
+        echo "linux-openssl-$parsed_version-arm64"
+      else
+        echo "linux-openssl-$parsed_version-amd64"
```

Comment:
There _shouldn't_ be an issue with changing the binary name here, right? We could consider following the convention you established with `macOS` where only the arm binary name contains the arch. But the mac case is probably special in that the x86 days are numbered. I'd expect arm64 and amd64 binaries to co-exist in the linux space pretty much indefinitely, so keeping a consistent `linx-openssl-platform` binary name seems like a reasonable approach.

And it makes it trivially easy to scale to something like armv8 for raspberry pi devices without further confusing the naming conventions.